_docs_integrate/attribute-introduction.md

```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
+#### Own shared and peer shared ThirdPartyRelationshipAttributes
 
-Sometimes a peer shared RelationshipAttribute is referred to as a peer shared ThirdPartyRelationshipAttribute. This is the case if it can be interpreted as a counterpart of an own shared ThirdPartyRelationshipAttribute. Note that the `shareInfo.sourceAttribute` property of a peer shared ThirdPartyRelationshipAttribute is always undefined, as the own shared RelationshipAttribute used as the source can only be available locally to its owner.
-{: .notice--info}
+Note that it is possible to share the underlying RelationshipAttribute of an [own shared RelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#own-shared-and-peer-shared-relationshipattributes) with peers who are not involved in the Relationship in which the RelationshipAttribute exists, provided that the `confidentiality` of the RelationshipAttribute is not `"private"`.
+The sharing of a RelationshipAttribute by its `owner` with such a peer in turn leads to the creation of another own shared RelationshipAttribute in the wallet of the `owner`, which contains the Address of this peer within its `shareInfo.peer` property.
+Such an own shared RelationshipAttribute is referred to as an own shared ThirdPartyRelationshipAttribute.
+As this own shared ThirdPartyRelationshipAttribute originates from another own shared RelationshipAttribute that exists in the context of a different Relationship, it contains the `id` of the source own shared RelationshipAttribute within its `shareInfo.sourceAttribute` property.
+This is the case as long as the own shared RelationshipAttribute used as the source has not been [deleted]({% link _docs_integrate/delete-attributes.md %}#delete-own-shared-attributes).
 
-Although the peer is not its `owner`, the underlying RelationshipAttribute of a peer shared RelationshipAttribute can be shared with peers who are not involved in the Relationship in which the RelationshipAttribute exists if its `confidentiality` is not `"private"`.
-The sharing of the RelationshipAttribute by the peer with another peer is technically documented by the creation of a so-called third party owned RelationshipAttribute.
+<div style="width: 640px; height: 720px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:720px" src="https://lucid.app/documents/embedded/68fbda7f-30c1-40d1-853a-fd94a91988e8" id="oSgEzd9YW_TQ"></iframe></div>
+
+In the wallet of the peer with whom the underlying RelationshipAttribute of the own shared RelationshipAttribute was shared, a corresponding peer shared RelationshipAttribute is created.
+It is also referred to as a peer shared ThirdPartyRelationshipAttribute.
```

Comment:
As above, I'd rephrase it to:
"In the wallet of the **third party**, with whom the underlying RelationshipAttribute of the own shared RelationshipAttribute was shared, a corresponding peer shared **ThirdPartyRelationshipAttribute** is created.

Reply:
I know what you mean and will take that into consideration. I think it depends on whether we want to treat "peer shared ThirdPartyRelationshipAttributes" as a special case of "peer shared RelationshipAttributes" or as a separate LocalAttribute class. What is your first opinion on this? We can probably only decide this once we have clarified the functionalities relating to ThirdPartyRelationshipAttributes in more detail.